flaskbb/utils/helpers.py

```diff
@@ -619,9 +621,20 @@ def __call__(self, environ, start_response):
 
 
 def real(obj):
-    """
-    Unwraps a werkzeug.local.LocalProxy object if given one, else returns the object
+    """Unwraps a werkzeug.local.LocalProxy object if given one,
```

Comment:
What's our line length? My yapf file is set to 100, but that's only because I copied it from work. Should we stick to 79/80?

Reply:
I have always tried to stick to 79/80 but in times with widescreen monitors I think that 100 would make sense :)